conftest.py

```diff
@@ -263,6 +263,12 @@ def local_sr_on_hostB1(hostB1):
     logging.info(">> local SR on hostB1 present : %s" % sr.uuid)
     yield sr
 
+@pytest.fixture(scope='session')
+def existing_shared_sr(host):
+    sr = host.pool.first_shared_sr()
+    assert sr is not None, "A shared SR on the pool is required"
+    return sr
```

Comment:
Changing the return to a yield will make the fixture more consistent with the others.